src/ast/ast_validations.c

```diff
@@ -1349,23 +1340,24 @@ references to outside variables");
 		for(uint i = 0; i < n_projections; i++) {
 			const cypher_astnode_t *proj =
 				cypher_ast_return_get_projection(return_clause, i);
-			const char *var_name;
-			const cypher_astnode_t *identifier =
+			const cypher_astnode_t *alias_node =
```

Comment:
are we using `alias_node` ?

Reply:
No, we don't need it. The variable was removed in  [23fa13d](https://github.com/RedisGraph/RedisGraph/pull/3107/commits/23fa13dd69f0528f3ca6b528f7b587ce54ba0569)
Thanks.